HIP/hip-904-frictionless-airdrop.md

```diff
@@ -8,232 +8,422 @@ type: Standards Track
-2. As an hts-token-airdrop-sender,  I want to airdrop a token to a receiver who has not associated with the token but has an open autoAssociation slot. I expect this should work exactly like the cryptoTransfer.
-3. As an hts-token-airdrop-sender,  I want to airdrop a token to a receiver who has not associated with the token and has no open autoAssociation slots. I expect this transfer will be marked as pending so that the receiver can claim at any time if it is still available at that time.
-4. As an hts-token-airdrop-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of an airdrop to an account that is already associated with a token to reduce the friction by a receiving account
-5. As an hts-token-airdrop-sender I want to cover the fees (custom, association & rent until the end of the first full auto renewal period) of an airdrop that consumes an automatic token slot to reduce the friction for a receiving account
-6. As an hts-token-airdrop-sender I want to cover the fees (airdrop, custom, association & rent until the end of the first full auto renewal period) of an airdrop that results in a pending transfer to reduce the friction by the intended receiving account.
-7. As an airdrop-sender I want to cancel pending token transfers that I initiated and are still unclaimed by the receiver.
-8. As a token treasury account holder, I want to airdrop tokens with custom fallback fees. I understand as the token treasure that custom fees are not accessed me and therefore the fallback fee will be non-applicable.
+1. As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has already
+   associated with the token. I expect this should work exactly like the crypto transfer.
+2. As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has not associated
+   with the token but has an open autoAssociation slot. I expect this should work exactly like the
+   cryptoTransfer.
+3. As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has not associated
+   with the token and has no open autoAssociation slots. I expect this transfer will be marked as
+   pending so that the receiver can claim at any time if it is still available at that time.
+4. As an hts-token-airdrop-sender I want to cover the custom fees (fixed, fractional, and royalty
+   excluding fallback) of an airdrop to an account that is already associated with a token to reduce
+   the friction by a receiving account
+5. As an hts-token-airdrop-sender I want to cover the fees (custom, association & rent until the end
+   of the first full auto renewal period) of an airdrop that consumes an automatic token slot to
+   reduce the friction for a receiving account
+6. As an hts-token-airdrop-sender I want to cover the fees (airdrop, custom, association & rent
```

Comment:
Isn't this use case already ingesting point 4. and 5. making them obsolete?

Reply:
A pending airdrop is created in this case, but is _not_ created in cases 4 and 5.  Case 4 is a simple immediate transfer, and case 5 is both an automatic association and immediate transfer.